modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -1,339 +1,425 @@
+You can manage load balancing with the tools connected to your infrastructure.
 
+* *Logging*
++
+Runtime Fabric uses Ops Center to view and configure logs. 
+You can use forward logs from Runtime Fabric to a rsyslog server.
++
+See xref:runtime-fabric::runtime-fabric-logs.adoc[View and Configure Logging in Runtime Fabric].
 
-== Deployment Strategy Flexibility
+* *Monitoring*
++
+Runtime Fabric provides dashboards and alerts in Ops Center for viewing performance and availability metrics. To receive and send alerts, use an SMTP server.
++
+See https://docs.mulesoft.com/runtime-fabric/1.4/configure-alerting.adoc[Configure Alerting on Runtime Fabric].
++
+You can also use Anypoint Monitoring to view application metrics with Anypoint Platform.
++
+See xref:runtime-fabric::manage-monitor-applications.adoc[Monitoring Applications Deployed to Runtime Fabric].
 
-If you want to deploy the same Mule application via various different deployment strategies, such as to an <<Cloud Console On-Premises Deployment, on-premises server>> and <<Cloud Console CloudHub Deployment, CloudHub>>, abstract some parameters of the application to xref:mule-runtime::intro-packaging.adoc[application properties] that you can set with different values in each use case, without needing to alter the actual application.
+* *Object store*
++
+Runtime Fabric does not support Object Store.
 
-Create an application properties file named *mule-app.properties* in the `src/main/app` folder of your project. When using the properties tab on CloudHub or Pivotal Cloud Foundry, these properties are overriden. See <<Managing Properties>> to see how these are loaded with values in each case.
+* *Scheduling*
++
+The scheduling feature is not available for Runtime Fabric deployments.
+To schedule tasks, use the Scheduler Endpoint element in your flows.
++
+See xref:mule-runtime::scheduler-concept.adoc[Scheduler Endpoint].
 
-== Using Different Environments
 
-The Anypoint Platform enables you to handle separate xref:access-management::environments.adoc[Environments], such as production, QA, Dev, or any other custom one you may want to create. You can set some of these to be _sandbox_ environments, which allows you the freedom to test and experiment away from the public eye.
+* *Security updates*
++
+When a security update is available for the runtime version for an app, you can see and apply the update in Runtime Manager.
 
-Regardless of the deployment strategy used, deployments are always done onto a specific environment. Each manages a different set of deployments, and accessing each requires a different set of xref:access-management::managing-permissions.adoc[permissions], allowing you to divide roles clearly between the different teams of your organization.
+* *Troubleshooting*
++
+Runtime Fabric does not include the Insight troubleshooting tool. 
+Instead, use the command-line tools provided to obtain the health status of the cluster and components.
```

Comment:
Given the above comment, this wouldn't be an "instead". This is about troubleshooting the Runtime Fabric itself.